### ISSUE TITLE
LAWS: Look Around and Warm-Start Natural Gradient Descent for Quantum Neural Networks

### ISSUE THREAD

**Issue by github-nakasho** [github-nakasho/quantph#2419]
# 論文概要

VQAでのBarren plateauを緩和するLAWS(Look Around Warm-Start)アルゴリズムを考案。モデルパラメータ初期化と量子自然勾配の高速収束を利用したもの。次の反復でのパラメータ更新のために、パラメータ探索空間を繰り返し再初期化する。

# 論文を理解する上で重要な図など

<img width="350" alt="Screen Shot 2022-05-06 at 9 35 29 AM" src="https://user-images.githubusercontent.com/32871721/167185997-f3009b2f-45cf-4d09-a0ab-a9be78b601e6.png">
<img width="355" alt="Screen Shot 2022-05-06 at 9 35 39 AM" src="https://user-images.githubusercontent.com/32871721/167186008-88e1498a-415f-490e-bd96-294730c459bf.png">
<img width="359" alt="Screen Shot 2022-05-06 at 9 35 49 AM" src="https://user-images.githubusercontent.com/32871721/167186014-ceb0655b-08fa-4287-a844-a7951214fcbc.png">
<img width="697" alt="Screen Shot 2022-05-06 at 9 36 02 AM" src="https://user-images.githubusercontent.com/32871721/167186019-2a8538e4-a2a8-42c1-8802-ca69ffc9529d.png">

# 論文リンク

[https://arxiv.org/abs/2205.02666](https://t.co/eJmuHyTwqH)
